conf/manifest.yaml.example

```diff
@@ -0,0 +1,31 @@
+MANIFESTER:
```

Comment:
`MANIFEST` here - the path that gets established in the `Settings` instance is dependent on the top level of the map, not the yaml file itself.

Our convention is for the file's top level map key to match the file name.